files/machine_provision.yml

```diff
@@ -89,6 +89,11 @@
 
                         if [ "${machine_type}" == "qdc" ]; then
                             machine_subarch=GrubWithOptionEfiboot
+                            if [ "${kernel_desc}" == "CentOS ERP 99" ]]; then
```

Comment:
if ${os_type} == centos is better here.

Reply:
if os_type was a variable in this job, yes it'd be better.

Reply:
Then this is better changed in the OS job?